### DISCUSSION TITLE
Fetching remote config values when the server starts

### DISCUSSION THREAD

**Discussion by JithinKS97** [vercel/next.js#14117] [Help]
I have a requirement in which I need to fetch some config values from a remote server when the application starts. This will contain the config values required for the app. Is it possible to do this and what is the best approach for this.

**Comment by rafaelalmeidatk** [comment]
What kind of config are you trying to fetch?

**Reply by JithinKS97** [reply]
The main config value is the api url. We have dev and prod environments. The api urls are different for these.
For example
```
config = {
  api: dev-api@example.com 
}
```
In the code we will be using like ```fetch(`${config.api/resource`)```
In the case of prod, config will be 
```
config = {
  api: prod-api@example.com 
}
```
